app/representers/api/v1/exercises/representer.rb

```diff
@@ -49,14 +49,6 @@ class Representer < BaseRepresenter
                  required: true
                }
 
-    collection :versions,
-             type: Array,
-             writeable: false,
-             readable: true,
-             getter: ->(user_options:, **) do
-               visible_versions(can_view_solutions: SOLUTIONS.call(user_options: user_options))
-             end
```

Comment:
Leave it in so as not to make a breaking API change?  If take out, need to double check that CNX isn't using this.

Reply:
I will check with Ross. But this has to wait for FE anyway.

Reply:
Ross says @philschatz wrote the CNX code. But the thinks it's not used.